src/providers/ContentShareProvider/docs/useContentShareControls.mdx

```diff
@@ -13,6 +13,17 @@ The `useContentShareControls` hook returns the state and functionality around st
   // Whether or not the local content share is paused
   paused: boolean;
 
+  // Whether or not the local user is currently sharing content
+  isLocalUserSharing: boolean;
```

Comment:
Can these be accessed from `useContentShareState` and we do not add them in this hook as well? The idea seems like this provides actions on content share vs the other one providing the state. I do see we have `paused` here, but checking if the other hook works or we need them repeated at two places.

Reply:
These are already available in this hook, we simply forgot updating the type definition and documentation 5 years ago. The only new property is `canStartContentShare`